cmd/cluster-config-operator/main.go

```diff
@@ -1,37 +1,21 @@
 package main
 
 import (
-	goflag "flag"
-	"fmt"
-	"math/rand"
 	"os"
-	"time"
 
 	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
 
-	utilflag "k8s.io/component-base/cli/flag"
-	"k8s.io/component-base/logs"
+	"k8s.io/component-base/cli"
 
 	"github.com/openshift/cluster-config-operator/pkg/cmd/operator"
 	"github.com/openshift/cluster-config-operator/pkg/cmd/render"
 	"github.com/openshift/cluster-config-operator/pkg/version"
 )
 
 func main() {
-	rand.Seed(time.Now().UTC().UnixNano())
-
-	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
-	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
-
-	logs.InitLogs()
```

Comment:
the new library appears to lose log initialization from before the flag parse.

Reply:
Right, I might have misunderstood if that's a desired behavior, given this line comment https://github.com/kubernetes/component-base/blob/master/logs/logs.go#L170 in the `InitLogs()`

Reply:
reverted back start command to simplify PR